src/integration-test/groovy/au/org/ala/fieldcapture/OrganisationReportingSpec.groovy

```diff
@@ -0,0 +1,253 @@
+package au.org.ala.fieldcapture
+
+import com.icegreen.greenmail.util.GreenMail
+import com.icegreen.greenmail.util.GreenMailUtil
+import pages.Organisation
+import pages.ReportPage
+import pages.ViewReportPage
+import spock.lang.Shared
+
+import javax.mail.internet.MimeMessage
+
+class OrganisationReportingSpec extends StubbedCasSpec {
+
+    @Shared
+    GreenMail greenMail = new GreenMail()
+
+    def orgId = "test_organisation"
+
+    def setupSpec(){
+        useDataSet("dataset_mu")
+        greenMail.start()
+    }
+
+    def cleanup() {
+        waitFor {
+            logout(browser)
+        }
+        greenMail.stop()
+
+    }
+
+    def "Organisation reports are displaying in the reporting tab"() {
+
+        setup:
+        loginAsMeritAdmin(browser)
+
+        when:
+        to Organisation, orgId
+        displayReportsTab()
+
+        then:
+        reportsTabPane.reports.size() > 0
+
+    }
+
+    def "We can specify the core services reporting frequency"() {
+        setup:
+        loginAsMeritAdmin(browser)
+
+        when: "Display the reporting tab"
+        to Organisation, orgId
+
+        then:
+        waitFor 60, {at Organisation}
+
+        waitFor 60, {
+            adminTab.click()
+        }
+
+        then:
+        waitFor 20, { adminTabContent.displayed }
+
+        when:
+        def reportingSection = adminTabContent.viewReportingSection()
+//        reportingSection.coreServicesGroup = "Quarterly (First period ends 30 September 2023)"
+        reportingSection.coreServicesGroup = "Quarterly - A (First period ends 31 March 2023)"
+
+        reportingSection.saveReportingGroups()
+
+        then:
+        waitFor 20,{
+            hasBeenReloaded()
+        }
+
+        when:
+        displayReportsTab()
+
+        then:
+        reportsTabPane.reports.size() > 0
+
+    }
+
+    def "A user with the admin role can complete Organisation reports,and submit them"() {
+        setup:
+        loginAsMeritAdmin(browser)
```

Comment:
Note the spec description is "a user with the admin role"... the organisation admin needs to be the user for this spec as they are user who will be filling in the reports in the live system.